ethereum-types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ethereum-types"
-version = "0.7.0"
+version = "0.8.0"
```

Comment:
note, that we haven't published ethereum-types 7.0, so no need to version bump here